qiskit_runtime/result/result.py

```diff
@@ -0,0 +1,65 @@
+# This code is part of qiskit-runtime.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Runtime result class"""
+
+import json
+from qiskit.result import Result
+from qiskit.result.postprocess import _hex_to_bin
+from qiskit_runtime.classes import QuasiDistribution
+
+
+class RuntimeResult(Result):
+    """A result class for the Qiskit runtime.
+    """
+    @classmethod
+    def decode(cls, data):
+        """Decoding for results from Qiskit runtime jobs.
+        """
+        return cls.from_dict(json.loads(data))
+
+    def get_quasiprobabilities(self, experiment=None):
+        """Get quasiprobabilites associated with one or more experiments.
+
+        Parameters:
+            experiment (int or list): Indices of experiments to
+                                      grab quasiprobabilities from.
+
+        Returns:
+            QuasiDistribution: Distribution for a single-experiment.
+            list: List of multiple distributions for requested list of experiments.
```

Comment:
The return value is a `QuasiDistribution` or a list of `QuasiDistribution`, right? This docstring is a bit ambiguous.

Reply:
Hopefully resolved this.

Reply:
I think you forgot to push the fix for this and the `quasi.py` docstring (still says mthree)

Reply:
OK, did it for real this time.